cmd/device-worker/main.go

```diff
@@ -145,8 +146,24 @@ func main() {
 
 	reg := registration2.NewRegistration(&hw, deviceOs, dispatcherClient, configManager, hbs, wl, dataMonitor, metricsStore, systemMetricsWatcher)
 
+	// defined how to connect to hosts
+	ansiblePlaybookConnectionOptions := &options.AnsibleConnectionOptions{ //FIXME
+		Connection: "local",
+		User:       "test_user",
```

Comment:
should we add this use to the blueprint so each edge device created will have a user name `test_user` ?

Reply:
I think that user is not needed for connection=local, is it?

Reply:
I'm unsure about what it is the right configuration.
@machacekondra you're right, connection local doesn't require either ssh or username.

my doubts here are: can a playbook connect to remote host? Should the connection type be only local? Otherwise how to configure these parameters?

Reply:
and, if the connection is on test_user, how can install a new package for example? Should we add that to sudo, no?